docs/docs/20-quickstart.mdx

```diff
@@ -1,11 +1,12 @@
 ---
-description: Kargo Quickstart
+description: Find out how to quickstart Kargo and progress changes through multiple stages byinteracting with your GitOps repository and Argo CD Application resources
```

Comment:
```suggestion
description: Find out how to quickstart Kargo and progress changes through multiple stages by interacting with your GitOps repository and Argo CD Application resources
```

> Find out how to quickstart...

The use of "quickstart" as a verb feels awkward to me. Would you be open to rewording this?

Reply:
Feel free to change 'quickstart' into 'set up' or any other similar verb ;)